src/rules/noUnusedVariableRule.ts

```diff
@@ -56,8 +56,9 @@ class NoUnusedVariablesWalker extends Lint.RuleWalker {
         if (!Lint.hasModifier(node.modifiers, ts.SyntaxKind.ExportKeyword)) {
             const importClause = node.importClause;
 
-            // named imports & namespace imports handled by other walker methods
-            if (importClause.name != null) {
+            // Named imports & namespace imports handled by other walker methods.
```

Comment:
fixed all code comments in https://github.com/palantir/tslint/pull/470. revert this comment to its original format. same thing in noUseBeforeDeclareRule.